lib/api/controllers/documentController.js

```diff
@@ -300,20 +328,20 @@ function DocumentController (kuzzle) {
    * @param {Request} request
    * @returns {Promise<Object>}
    */
-  this.mDelete = function documentMDelete (request) {
-    let promises = [];
+  mDelete(request) {
+    const promises = [];
 
     assertHasBody(request);
     assertBodyHasAttribute(request, 'ids');
     assertBodyAttributeType(request, 'ids', 'array');
     assertHasIndexAndCollection(request);
 
-    if (request.input.body.ids.length > kuzzle.config.limits.documentsWriteCount) {
-      throw new BadRequestError(`Number of delete to perform exceeds the server configured value (${kuzzle.config.limits.documentsWriteCount})`);
+    if (request.input.body.ids.length > this.kuzzle.config.limits.documentsWriteCount) {
+      throw new BadRequestError(`Number of delete to perform exceeds the server configured value (${this.kuzzle.config.limits.documentsWriteCount})`);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 150 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)